sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -828,6 +828,19 @@ class SubquerySuite extends QueryTest with SharedSQLContext {
           |               from t2 lateral view explode(arr_c2) q as c2
                           where t1.c1 = t2.c1)""".stripMargin),
         Row(1) :: Row(0) :: Nil)
+
+      val msg1 = intercept[AnalysisException] {
+        sql(
+          """
+            | select c1
+            | from t2
+            | where exists (select *
+            |               from t1 lateral view explode(t2.arr_c2) q as c2
+            |               where t1.c1 = t2.c1)
```

Comment:
@gatorsmile The other test was in lower case . so i had it in lower case for consistency. I will change both the SQLs in this test. Thank you.